WordPressAuthenticator/Unified Auth/View Related/SiteAddressViewController.swift

```diff
@@ -1,28 +1,32 @@
 import UIKit
+import WordPressUI
 
 
 /// SiteAddressViewController: log in by Site Address.
 ///
 final class SiteAddressViewController: LoginViewController {
 
+    /// Private properties
+    ///
     @IBOutlet private weak var tableView: UITableView!
     @IBOutlet var bottomContentConstraint: NSLayoutConstraint?
 
     // Required property declaration for `NUXKeyboardResponder` but unused here.
     var verticalCenterConstraint: NSLayoutConstraint?
 
-    var displayStrings: WordPressAuthenticatorDisplayStrings {
+    private var displayStrings: WordPressAuthenticatorDisplayStrings {
```

Comment:
this won't really be private as the singleton isn't. I wonder if we could just reference directly to the property of the singleton since it's used in just a couple of places?

Reply:
Just another option - if you're goal is to just have a shorter name, you could use `typealias`. 

`typealias DisplayStrings = WordPressAuthenticator.shared.displayStrings`

Reply:
I forgot about using a typealias. Thanks!

Reply:
`typealias` doesn't work with singletons. Error: `Static property 'shared' is not a member type of 'WordPressAuthenticator'`. I'll just reference the properties directly.